### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by Seldaek**

### CODE REVIEW

File: .github/workflows/functional.yml [hunk 1]
```diff
@@ -35,29 +35,16 @@ jobs:
...
+      - uses: actions/checkout@v4
 
-      - name: Install PHP
-        uses: shivammathur/setup-php@v2
+      - uses: shivammathur/setup-php@v2
         with:
-          coverage: xdebug
-          php-version: ${{ matrix.php-version }}
+          php-version: "${{ matrix.php-version }}"
+          coverage: none
```
Comment (johnstevenson) [inline .github/workflows/functional.yml:43]: This causes all functional tests to be skipped! Should be `coverage: xdebug`

Comment (Seldaek) [reply .github/workflows/functional.yml:43]: Sorry missed that in my copy pasta party.. Fixed

**Comment by Seldaek** [comment]
Btw I did the same changes on all composer repos, so just bringing stuff in line with each other and a bit of modernization

**Comment by johnstevenson** [comment]
Thanks for that. And the dependabot commit.

**Pull request merged**